src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```

read enum naming convention, it should be singular